src/cli/import.rs

```diff
@@ -120,9 +147,28 @@ async fn import_conda_env(args: Args) -> miette::Result<()> {
 
     // TODO: add dry_run logic to import
 
-    let file = CondaEnvFile::from_path(&file)?;
-    let (feature_string, environment_string) =
-        get_feature_and_environment(&args.feature, &args.environment, &file)?;
+    let (env_file, feature_string, environment_string) = match format {
```

Comment:
```rust
    struct ImportFileOutput {
        CondaEnv(EnvFile, String, String),
        PyPiTxt(String, String)
    }
```

Let's use an enum instead of the tuple as return value of that expression.
Maybe with a better name than `ImportFileOutput` :smile: